index.html

```diff
@@ -56,6 +56,148 @@ <h2>I'm Cristhian. Glad to see you!</h2>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+                </ul>
+              </div>
+              <button type="button">See project</button>
+            </div>
+          </div>
+          <div class="card">
+            <img class="img-project" src="/img/Snapshoot-Portfolio3.svg" alt="">
+            <div class="description-project tonic">
+              <h3>Tonic</h3>
+              <div class="frame">
+                <p>CANOPY</p>
+                <img src="/img/Counter.svg">
+                <p>Back End Dev</p>
+                <img src="/img/Counter.svg">
+                <p>2015</p>
+              </div>
+              <p class="text-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="technologys">
+                <ul class="list-technologys">
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+                </ul>
+              </div>
+              <button type="button">See project</button>
+            </div>
+          </div>
+          <div class="card">
+            <img  class="img-project" src="/img/Snapshoot-Portfolio4.svg" alt="">
+            <div class="description-project multi">
+              <h3>Multi-Post Stories</h3>
+              <div class="frame">
+                <p>CANOPY</p>
+                <img src="/img/Counter.svg">
+                <p>Back End Dev</p>
+                <img src="/img/Counter.svg">
+                <p>2015</p>
+              </div>
+              <p class="text-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="technologys">
+                <ul class="list-technologys">
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+                </ul>
+              </div>
+              <button type="button">See project</button>
+            </div>
+          </div>
+        </div>
+        </div>
+      </section>
+      <section class="section-p3">
+        <div class="about-me">
+          <div class="description1">
+            <h2>About Myself</h2>
+            <p>
+              I'm a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you like
+              what you see and have a project you need coded, don't hesitate to
+              contact me.
+            </p>
```

Comment:
- [ ] kindly consider adding the lets connect link which is under this paragraph in the design to implement exactly the given design

Reply:
Sorry. Can you explain this more.